### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by dai-shi**
for v4.5.5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -203,15 +203,15 @@
   "homepage": "https://github.com/pmndrs/zustand",
   "packageManager": "pnpm@8.15.0",
   "dependencies": {
-    "use-sync-external-store": "1.2.0"
+    "use-sync-external-store": "1.2.2"
```
Comment (dai-shi) [inline package.json:206]: I didn't check the changelog. Is there a changelog somewhere?

Comment (charkour) [reply package.json:206]: Seems like there was only a version bump but no source code changes: https://npmdiff.dev/use-sync-external-store/1.2.0/1.2.2/

Comment (charkour) [reply package.json:206]: I checked with Renovate too and found the same change:

https://app.renovatebot.com/package-diff?name=use-sync-external-store&from=1.2.0&to=1.2.2

Comment (dai-shi) [reply package.json:206]: Thanks for checking!!

**Pull request merged**